www/src/components/layout/page-heading.js

```diff
@@ -0,0 +1,88 @@
+import React from "react"
+import PropTypes from "prop-types"
+import styled, { css } from "react-emotion"
+
+import presets, { colors } from "../../utils/presets"
+import { rhythm, options, scale } from "../../utils/typography"
+
+const PageHeadingContainer = styled.header({
+  padding: rhythm(options.blockMarginBottom),
+
+  [presets.Tablet]: {
+    left: 0,
+    position: "fixed",
+    padding: 0,
+    top: `calc(${presets.bannerHeight} + ${presets.headerHeight})`,
+  },
+})
+
+const H1 = styled.h1({
+  alignItems: "center",
+  color: colors.lilac,
+  display: "flex",
+  fontSize: "1.5rem",
+  margin: 0,
+  position: "relative",
+  width: "100%",
+
+  [presets.Tablet]: {
+    transform: "rotate(-90deg) translate(calc(-100% - 2rem), 0.7rem)",
+    transformOrigin: "top left",
+  },
+
+  "&:after": {
+    bottom: "2rem",
+    content: "attr(data-title)",
+    display: "none",
+    fontSize: "12rem",
+    opacity: "0.03",
+    position: "absolute",
+    right: "-0.7rem",
+    zIndex: "-1",
+
+    [presets.Tablet]: {
+      display: "block",
+    },
+  },
+})
+
+const Icon = styled.span({
+  display: "block",
+  width: "36px",
+  height: "32px",
+  margin: "0.1rem 0.1rem 0 - 0.3rem",
+
+  "& .svg-stroke": {
+    strokeMiterlimit: 10,
+    strokeWidth: 1.4173,
+  },
+  "& .svg-stroke-accent": { stroke: colors.lavender },
+  "& .svg-stroke-lilac": { stroke: colors.lavender },
+  "& .svg-fill-lilac": { fill: colors.lavender },
+  "& .svg-fill-gatsby": { fill: colors.lavender },
+  "& .svg-fill-brightest": { fill: `#fff` },
+  "& .svg-fill-accent": { fill: colors.lavender },
+  "& .svg-stroke-gatsby": { stroke: colors.lavender },
+  "& .svg-fill-gradient-accent-white-top": { fill: `transparent` },
+  "& .svg-fill-gradient-accent-white-45deg": { fill: `transparent` },
+  "& .svg-fill-gradient-accent-white-bottom": { fill: `#fff` },
+  "& .svg-fill-gradient-purple": { fill: colors.lavender },
+  "& .svg-stroke-gradient-purple": { stroke: colors.lavender },
+  "& .svg-fill-wisteria": { fill: `transparent` },
```

Comment:
Same here — no need for the `&` unless you have specificity problems.

Reply:
OK